### DISCUSSION TITLE
The colors of page content , sidebar and page header are the same.

### DISCUSSION THREAD

**Discussion by allyson-silva** [saadeghi/daisyui#2952] [Ideas / feature requests]
The page header and the side menu always have the same color, making it impossible to differentiate between them regardless of the theme used. I suggest that the top bar have a slightly darker tone than the theme's default for better differentiation. Additionally, the content should also have a different tone to create a visual hierarchy and improve readability. For example, the content could have a lighter tone than the top bar, or it could use a different color altogether.
Example:
![admin lte](https://github.com/saadeghi/daisyui/assets/28867801/67473872-d71a-400d-a4ef-d319733cd9c9)

**Comment by saadeghi** [comment]
They don't have colors by default.  
You can add any color to them like `bg-base-200` or `bg-base-300` or even Tailwind color names if you want.

Let me know if you have a question.
